daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -359,6 +359,43 @@ function Set-Repos {
   Run-Command 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' addrepo 'google-compute-engine-stable' 'https://packages.cloud.google.com/yuck/repos/google-compute-engine-stable'
 }
 
+function Export-ImageMetadata {
+  $version = (Get-ComputerInfo).OsVersion
+  $family = 'windows-' + (Get-ComputerInfo).windowsversion
+  $name =  (Get-ComputerInfo).OSName
+  $release_date = (Get-Date).ToUniversalTime()
+  $image_metadata = @{'family' = $family;
+                      'version' = $edition;
+                      'name' = $name;
+                      'location' = ${script:outs_dir};
+                      'build_date' = $release_date;
+                      'build_repo' = $stage;
+                      'packages' = @()}
+
+  # Get Googet packages.
+  $out = & 'C:\ProgramData\GooGet\googet.exe' -root 'C:\ProgramData\GooGet' 'installed'
+  $out = $out[1..$out.length]
+  [array]::sort($out)
+
+  foreach ($package_line in $out) {
+    $split = $package_line.Trim() -split '\s+'
+    $name = $split[0]
+    $version = $split[1]
+    # TODO: Currently, "Googet Installed" command only return name and version.
+    # We need to update googet command to get all info.
```

Comment:
If you haven't already, can you add an issue in the Googet repo and reference it here?

Reply:
This already exists, use the -info flag

Reply:
@adjackura i am not aware of this. should i remove this TODO.

Reply:
Will fix.